src/core/utils/resultsutils.js

```diff
@@ -0,0 +1,17 @@
+import SearchStates from '../storage/searchstates';
+
+/**
+ * Returns a CSS class for the input searchState
+ * @param {string} searchState - A search state from {@link SearchStates}
```

Comment:
can this type be `SearchStates`

Reply:
Isn't searchStates the entire object while the individual searchState is the string? I originally had it this way but my jsdoc yells at me it isn't a string.